tests/dp_computations_test.py

```diff
@@ -0,0 +1,132 @@
+import unittest
+import numpy as np
+from scipy.stats import skew, kurtosis
+
+import pipeline_dp
+from pipeline_dp.dp_computations import *
+
+
+class MeanVarParams(unittest.TestCase):
+    def test_l0_sensitivity(self):
+        params = pipeline_dp.dp_computations.MeanVarParams(eps=1, delta=1e-10, low=2, high=3,
+                                                           max_partitions_contributed=4,
+                                                           max_contributions_per_partition=5,
+                                                           noise_kind=pipeline_dp.NoiseKind.LAPLACE)
+        self.assertEqual(params.l0_sensitivity(), 4)
+
+    def test_l1_sensitivity(self):
+        self.assertEqual(pipeline_dp.dp_computations.compute_l1_sensitivity(l0_sensitivity=4,
+                                                                            linf_sensitivity=12),
+                         48)
+
+    def test_l2_sensitivity(self):
+        self.assertEqual(pipeline_dp.dp_computations.compute_l2_sensitivity(l0_sensitivity=4,
+                                                                            linf_sensitivity=12),
+                         24)
+
+    def test_linf_sensitivity(self):
+        params = pipeline_dp.dp_computations.MeanVarParams(eps=1, delta=1e-10, low=2, high=3,
+                                                           max_partitions_contributed=4,
+                                                           max_contributions_per_partition=5,
+                                                           noise_kind=pipeline_dp.NoiseKind.LAPLACE)
+
+        # COUNT aggregation
+        self.assertEqual(params.linf_sensitivity(metric=pipeline_dp.Metrics.COUNT), 5)
+
+        # SUM aggregation
+        # Positive low, positive high
+        self.assertEqual(params.linf_sensitivity(metric=pipeline_dp.Metrics.SUM), 15)
+
+        # Negative low, positive high
+        params.low = -2
+        self.assertEqual(params.linf_sensitivity(metric=pipeline_dp.Metrics.SUM), 15)
+
+        # Negative low, negative high
+        params.high = -1
+        self.assertEqual(params.linf_sensitivity(metric=pipeline_dp.Metrics.SUM), 10)
+
+    def test_compute_sigma(self):
+        self.assertEqual(
+            pipeline_dp.dp_computations.compute_sigma(eps=1, delta=1, l2_sensitivity=10),
+            np.sqrt(2 * np.log(1.25)) * 10)
+        self.assertEqual(
+            pipeline_dp.dp_computations.compute_sigma(eps=0.5, delta=1e-10, l2_sensitivity=10),
+            np.sqrt(2 * np.log(1.25 / 1e-10)) * 20)
+
+    def _test_laplace_noise(self, results, value, eps, l1_sensitivity):
+        self.assertAlmostEqual(np.mean(results), value, delta=0.075)
```

Comment:
please change 0.075 -> 0.1 everywhere. It's better to keep things simpler. Moreover it should decrease chances of failng to less <1%